mmv1/products/firebasestorage/api.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: FirebaseStorage
+display_name: Cloud Storage for Firebase
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://firebasestorage.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+  - !ruby/object:Api::Resource
+    name: "Bucket"
+    min_version: beta
+    base_url: projects/{{project}}/buckets
+    self_link: projects/{{project}}/buckets/{{bucket_id}}
+    create_url: projects/{{project}}/buckets/{{bucket_id}}:addFirebase
+    delete_url: projects/{{project}}/buckets/{{bucket_id}}:removeFirebase
+    delete_verb: :POST
+    description: A storage bucket and its relation to a parent Firebase project.
```

Comment:
From this description I haven't gained an understanding of exactly what this resource is. Could you elaborate here on what a firebase storage bucket is, and perhaps how it's used in relation to other firebase resources.

Reply:
Added more description. It basically annotates a Google Cloud Storage bucket as being used by Firebase.